mtp_api/apps/notification/management/commands/send_notification_report.py

```diff
@@ -319,13 +317,13 @@ def serialise(self, worksheet, record: Disbursement, triggered: Triggered):
             'Prisoner name': record.prisoner_name,
             'Prison': record.prison.short_name,
             'Recipient name': record.recipient_name,
-            'Payment method': str(payment_method),
+            'Payment method': str(DisbursementMethod[record.method].label),
```

Comment:
I guess the record is in the DB and it _should_(?) only have one of the valid enum values so we can safely use `[]`.

Reply:
yup!